### PR TITLE
Update Process Workflow with Gherkin

### PR DESCRIPTION
**Pull request by garaujodev**
## Motivation

We are using a ClubHouse template now, that follows Gherkin BDD workflow

## Proposed Solution
 - [x] Add information about template

### CODE REVIEW

File: resources/process_workflow.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ Be sure not to create a task such as "create a input field", or "create an API",
 
 All SMART tasks should be linked to a User Story, which should be properly detailed with its goal (it can be written using the User Story pattern), Acceptance criteria, and BDD if necessary.
 
+We are following Gherkin BD workflow to describe our User Stories, so, we have a template in ClubHouse to be used:
```
Comment (wevtimoteo) [inline resources/process_workflow.md]: **suggestion**: fix typo:

```suggestion
We are following Gherkin syntax to describe our User Stories:
```

There is no need to specify that its in `Clubhouse`

File: resources/process_workflow.md [hunk 2]
```diff
@@ -35,6 +35,19 @@ Be sure not to create a task such as "create a input field", or "create an API",
...
+
+*    Title: [Verb] [What] [Where]:
```
Comment (wevtimoteo) [inline resources/process_workflow.md]: **suggestion:** add markdown in code format:

```
<!-- Template markdown here -->
```

**Review by wevtimoteo — COMMENTED** [review]
Consider my comments then I'll review it again

File: resources/process_workflow.md (hunk 2 above)
Comment (garaujodev) [reply resources/process_workflow.md]: Done in b6c62c3

File: resources/process_workflow.md (hunk 1 above)
Comment (garaujodev) [reply resources/process_workflow.md]: Done in 7e85115

File: resources/process_workflow.md [hunk 3]
```diff
@@ -35,6 +35,28 @@ Be sure not to create a task such as "create a input field", or "create an API",
 
 All SMART tasks should be linked to a User Story, which should be properly detailed with its goal (it can be written using the User Story pattern), Acceptance criteria, and BDD if necessary.
 
+We are following Gherkin syntax to describe our User Stories:
+
+### Title:
+```
```
Comment (wevtimoteo) [inline resources/process_workflow.md:42]: **suggestion:** you should add a linebreak after a Markdown title

Review other occurrences

File: resources/process_workflow.md [hunk 4]
```diff
@@ -35,6 +35,28 @@ Be sure not to create a task such as "create a input field", or "create an API",
...
+
+### Description:
+```
+- Scenario:
+- Given
+- And
+- When
+- Then
+
```
Comment (wevtimoteo) [inline resources/process_workflow.md]: **suggestion:** removing this additional empty line

**Review by wevtimoteo — COMMENTED** [review]
Consider my comments then I'll review it again

File: resources/process_workflow.md (hunk 4 above)
Comment (garaujodev) [reply resources/process_workflow.md]: Done in ba05e0e

File: resources/process_workflow.md (hunk 3 above)
Comment (garaujodev) [reply resources/process_workflow.md:42]: Done in Done in ba05e0e

**Review by wevtimoteo — APPROVED** [review]

**Review by beatrizChagas — APPROVED** [review]
Good job!

**Pull request merged**